core/tests/org.openjdk.jmc.flightrecorder.rules.jdk.test/baseline/JfrRuleBaseline.xml

```diff
@@ -0,0 +1,4997 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Should be excluded still, I think?

Reply:
It should at least be added in a separate issue. ;)

Reply:
Indeed!